tests/resources/splash/wlo-svg-logo.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
What is this file for?

Reply:
Oh, good catch. Was used for mocking in `test_extract_non_html_content` but now the tests does use the real playwright container. 

Removed it